app/app.js

```diff
@@ -9,12 +10,38 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true}));
 app.use(express.static(path.join(__dirname, '..', 'public')));
 
-/*
-=====
-OTHER ROUTES GO HERE
-=====
-*/
+app.get('/api/formdata', function(req, res) {  // Will return all entries
+  Entry.find()
+  .then(function(entry) {
+    res.status(200).json(entry)
+  })
+  // curl --request GET 'localhost:3000/api/formdata'
+})
 
+app.post('/api/formdata', function(req, res) {
+  var entry = new Entry({
+    userId: req.body.userId,
```

Comment:
this will be req.user._id from the deserialized user object, see line 29 of app.js in healthme code for an example

userid won't be sent from the frontend